addons/dashboard/2.0.x/dashboard-1.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: dashboard
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "2.0.0-beta6"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "2.0.0-1"
```

Comment:
This sticks out to me, moving from beta6 to 1. Just mentioning here to make sure this is what we want.